README.md

```diff
@@ -1625,9 +1627,10 @@ Now a bit on how these fit together:
   taxpayers never worry about.
   Generally, you do not pay AMT unless it is “triggered” by specific situations, typically
   [high income (>$250K) or high deductions](http://www.marketwatch.com/story/congratulations-you-owe-the-alternative-minimum-tax-2014-01-14).
-  It also depends on where you file, since your state taxes can significantly affect your
-  deductions. Confusingly, if you are affected, AMT tax rates are usually at **26%** or **28%**
-  marginal tax rate, but effectively **35%** for some ranges, meaning it is
+  Whether you pay AMT also depends on where you file, since your state taxes can
+  significantly affect your deductions.
+  Confusingly, if you are affected, AMT tax rates are usually at **26%** or **28%** marginal tax
```

Comment:
Could omit "Confusingly". The statement is pretty clear.

Reply:
Fixed! I hate adverbs!